api4/post.go

```diff
@@ -195,12 +195,39 @@ func getFlaggedPostsForUser(c *Context, w http.ResponseWriter, r *http.Request)
 		posts, err = c.App.GetFlaggedPosts(c.Params.UserId, c.Params.Page, c.Params.PerPage)
 	}
 
+	pl := model.NewPostList()
+	var channelsAllowed []string
+
+	for _, post := range posts.Posts {
+		allowed := false
+
+		for _, channel := range channelsAllowed {
```

Comment:
Why a slice instead of a map?

Reply:
No particular reason - I guess in this case a `map[string]bool` would be cleaner - I'll update it.